lib/t_web/channels/feed_channel.ex

```diff
@@ -407,10 +408,19 @@ defmodule TWeb.FeedChannel do
   end
 
   def handle_info({Feeds, [:mode_change, event]}, socket) do
-    case event do
-      :start -> push(socket, "live_mode_started", %{})
-      :end -> push(socket, "live_mode_ended", %{})
-    end
+    socket =
+      case {event, socket.assigns[:mode]} do
+        {:start, "normal"} ->
+          push(socket, "live_mode_started", %{})
+          assign(socket, mode: "live")
+
+        {:end, "live"} ->
+          push(socket, "live_mode_ended", %{})
+          assign(socket, mode: "normal")
+
```

Comment:
`{event, ...}` is to avoid sending `live_mode_ended` to users who weren't even in the live event, for example, when newbies "live" ends, all `:mode_change` subscribers receive `[:mode_change, :end]` but only the newbies react to it (since only they are `"live"` in it).

Reply:
ah, this seem to solve my earlier comment above